json2html/jsonconv.py

```diff
@@ -78,88 +78,81 @@ def columnHeadersFromListOfDicts(self, inputtedJson):
                 -----------------------------
 
             @contributed by: @muellermichel
-        '''
+        """
 
-        if len(inputtedJson) < 2:
-            return None
-        if not isinstance(inputtedJson[0], dict):
+        if not inputted_json or not isinstance(inputted_json[0], dict):
```

Comment:
@softvar here is the functional change. Apologies it is hard to find amongst all the PEP8 changes - I should have made two separate commits. The issue was with the len(inputtedJson) < 2, whereas it should have been 1 to deal with the edge case. The final line is a nicer way to achieve the desired effect.